core-data/resolvers.js

```diff
@@ -44,3 +45,11 @@ export async function* getPostType( state, slug ) {
 	const postType = await apiRequest( { path: `/wp/v2/types/${ slug }?context=edit` } );
 	yield receivePostTypes( postType );
 }
+
+/**
+ * Requests theme supports data from the index.
+ */
+export async function* getThemeSupports() {
```

Comment:
Elsewhere we seem to be anticipating that index data may be broadly useful, rather than tracking only theme supports. But it's not consistent with how we've implemented resolvers and selectors, as there's only exposed access to theme supports. Scaling this out to other properties will require (maybe not so obviously) refactoring of the resolver to implement an `isFulfilled` for all index-sourced values (see #6084).

Which is to say: We should probably be consistent one way or the other: Only track theme supports, or implement resolvers and selectors as broadly index-sourced.

Reply:
I think it's better to focus specifically on theme supports data: 1c6e6936